python/tvm/rpc/client.py

```diff
@@ -397,15 +421,41 @@ def connect(url, port, key="", session_timeout=0):
         the connection when duration is longer than this value.
         When duration is zero, it means the request must always be kept alive.
 
+    session_constructor: List
+        List of additional arguments to passed as the remote session constructor.
```

Comment:
well it's more like a list, the first arg is a string specifying the name of the constructor, and following args are the positional args to that function. update docstring?